AODN/AODN-WAVE-NRT/ARDC_API_NRT/ardc_nrt/config/sofar/sources_id_metadata.json

```diff
@@ -601,5 +601,113 @@
     "water_depth": 31,
     "water_depth_units": "m",
     "wave_buoy_type": "directional"
+  },
+ "SPOT-31574C":
+  {
+    "spotter_id": "SPOT-31574C",
+    "site_name": "Hillarys",
+    "latitude_nominal": -31.85125,
+    "longitude_nominal": 115.6464667,
+    "institution": "IMOS Coastal Wave Buoys",
+    "institution_code": "IMOS",  
+    "deployment_start_date": "2024-11-22T00:00:00.000Z",
+    "abstract": "Near real time integral wave parameters from wave buoys collected by IMOS using a SOFAR Spotter-V3 at Hillarys",
+    "title": "Near real time integral wave parameters from wave buoys collected by IMOS using a SOFAR Spotter-V3 at Hillarys",
+    "project": "IMOS",
+    "citation": "The citation to be used in publications using the dataset should follow the format: \"Integrated Marine Observing System, [year-of-data-downloaded], Wave buoys Observations - Australia - near real-time, Downloaded from [url] on [date-of-download]\"",
+    "instrument": "SOFAR Spotter-V3",
+    "instrument_burst_interval": 1800,
+    "instrument_burst_duration": 1800,
+    "instrument_burst_units": "s",
+    "instrument_sampling_interval": 0.4,
+    "platform": "moored surface buoy",
+    "wave_motion_sensor_type": "GPS",
+    "wave_sensor_serial_number": "SPOT-31574C",
+    "hull_serial_number": " ",
+    "transmission": "Iridium/Cellular",
+    "water_depth": 28.5,
+    "water_depth_units": "m",
+    "wave_buoy_type": "directional"
+  },
+ "SPOT-31574C":
```

Comment:
you can remove the Coral Bay deployment has it is already in the config under SPOT-31886C

Reply:
I have also noticed other issue, I missed last time I reviewed the config:
 1)SPOT-1476 - the spotted name is misspelled in the config, it should be "Indented Head". Also, it seems like it is not being processed
2) There is two Cape Bridgewater deployments in the config. But I think  Spot-31761C is the only active. Note: If there are multiple active deployments at a single site, we should ensure they are named distinctly to differentiate between them